Add additional info on Origins and Tracks - OpenGov

DrW3RK opened the pull request:
Brief description of the origin of a track

Will create a separate PR for the privileges of each origin

DrW3RK committed:
Add additional info on Origins and Tracks - OpenGov

DrW3RK committed:
Add descriptions

From https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/governance/origins.rs

alfarok committed:
link Origins outlined in table to description sub-sections below

alfarok commented on components/Referenda-Tracks.jsx:
```diff
@@ -57,7 +57,7 @@ async function GetTracks(network, wsUrl, setReturnValue) {
       <tr key={trackData[0]}>
         <td><b>{trackData[0]}</b></td>
-        <td><b>{origin}</b></td>
+        <td><a href={`#${origin.toLowerCase().replace(" ", "-")}`}><b>{origin}</b></a></td>
```
@DrW3RK this line should do the trick in linking to the descriptions provided below the table.

profradha replied:
Perfect! Thank you.

alfarok committed:
remove extra newlines

DrW3RK committed:
Update values to reflect the actual code

https://github.com/paritytech/polkadot/blob/30ce14b5cc71aa4bf5f96e00ca61d973020131ea/runtime/kusama/src/governance/origins.rs#L170
